### ISSUE TITLE
Wireless Redstone Transmitter block each other

### ISSUE THREAD

**Issue by TDudes** [PrinceOfAmber/Cyclic#1878]
Minecraft Version:  1.16.4

Forge Version: 35.1.36

Mod Version: 1.3.3

Single Player or Server: Single

Describe problem (what you were doing; what happened; what should have happened):
When you Wireless Redstone Transmitter are connected to a single Receiver the first Transmitter will no longer function but the second one will.

They will both keep the on look until the broken first one is switched again which will then turn off the second.

I don't know if this is a feature, but if not as both seem connected, As no error occur in Minecraft Logs, but as both stay on unless the bad one is pressed, I'm guessing that the double connection is making the second break the first.

**Comment by Lothrazar** [comment]
Yeah, that sounds like it just makes sense.  If one transmitter is pushing out an ON signal, the receiver will be on.  If another transmitter is also connected to the same reciever, but the second one is off,ok thats fine but its still on from the first one.

What is supposed to happen when one says on and one says off? it cant be both. so like (ON and OFF) = ON . idk how else it would resolve

**Comment by TDudes** [comment]
Sorry I wasn't clear on how I tested, I used button to send a pulse for On and Off as this would remove the chance of the "And"/"Or" values. As you stated with "And" or would be the opposite.

My test used buttons on 2 transmitters 1 dead black between with both connected to the same receiver.  I did this will multiple blocks setups to test and see if I could get this to function. (None did, and all acted the same)

By base setup was to have 2 GPS items set to a receiver (I've tried on multiple sides for the GPS all acted the same), and put them into the two transmitters. On the transmitter that had the GPS item given first will function normally till the second one is put in the second transmitter. It would be at this time that the second transmitter will work, while the first no longer causes a redstone signal and will stay in the "On" state. Once the second transmitter is used it will function as a pulse with the button, but will stay in the "On" state even though a signal isn't being sent until the first broken transmitter is used at which will turn off the first.

give me a little and I'll get a few pictures of the setup.

Here is a test setup I made with wool showing where the GPS items are transferring too. I found something interesting while setting this up at which I found that the broken transmitter does "Function" by sending a ghost signal, and doesn't activate the block or redstone. This only allows the redstone lamp to turn on, but not active anything else such as the second Transmitter.
![Testing Setup](https://user-images.githubusercontent.com/18519729/126758841-61eee2ae-cdb9-4dda-b302-a83b18cff13d.png)
![Testing Setup 2](https://user-images.githubusercontent.com/18519729/126758856-ef9c8f3e-8e41-4b93-aaf8-7c8700858b33.png)
